README.md

```diff
@@ -396,6 +396,24 @@ Generating GraalVM config files is a manual process currently. You need to run t
 
 These files should be placed in `native-server/src/main/resources/META-INF/native-image` and are automatically used by the native-image process.
 
+## Building multi-architecture image
+
+Currently, docker image supports one platform: `amd64`. To publish it for two different platforms: `amd64` and `arm64`, module `server` in `build.sbt` is modified as follows:
```

Comment:
after merging, "currently" will become "in the past". The readme should explain the current setup, not the history of changes - that's what e.g. commit messages, PR messages, release notes are for.

So here we need an explanation on how the build process works, how to set it up locally (do I need to install the buildx plugin somehow?), and how to build locally for a single architecture

Reply:
Ok, I modified readme